SETUP.md

```diff
@@ -104,6 +104,16 @@ TRAVIS_ENCRYPT_ID=`grep \
 sed --in-place "s/f2f00aaf6402/$TRAVIS_ENCRYPT_ID/g" deploy.sh
 ```
 
+For setup on MAC OSX using BSD sed use the following commands.
```

Comment:
We could try a python command like:

```sh
python -c "import re; print(re.search('encrypted_([a-zA-Z0-9]+)_key', open('README.md').read())).group(0)"
```

Not tested BTW.

Reply:
Test the python and it works. Updating branch with new commands.